recipes/fontconfig/all/conanfile.py

```diff
@@ -61,15 +61,13 @@ def requirements(self):
             self.requires("libuuid/1.0.3")
 
     def validate(self):
-        if microsoft.is_msvc(self) and scm.Version(self.version) < "2.13.93":
+        if microsoft.is_msvc(self):
             raise ConanInvalidConfiguration("fontconfig does not support Visual Studio for versions < 2.13.93.")
 
     def build_requirements(self):
         self.build_requires("gperf/3.1")
         self.build_requires("pkgconf/1.7.4")
-        if microsoft.is_msvc(self):
-            self.build_requires("meson/0.63.1")
-        elif self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
```

Comment:
```suggestion
        if self._settings_build.os == "Windows" and not self.conf.get("tools.microsoft.bash:path", check_type=str):
```
MSYS2 recipe is already prepared to use Conan conf.